src/org/apache/xerces/impl/Constants.java

```diff
@@ -332,6 +332,9 @@ public final class Constants {
     /** Feature to ignore errors caused by unparsed entities ("validation/unparsed-entity-checking") */
     public static final String UNPARSED_ENTITY_CHECKING_FEATURE = "validation/unparsed-entity-checking";
     
+    /** Feature to cause nodes to populate location (file, line, column) information. */
+    public static final String LOCATION_INFO_FEATURE = "dom/include-location-info";
```

Comment:
if someone is using this feature, they know they're using Xerces and can assume they can downcast to the implementation class, so this all makes sense.